config/envs/default.py

```diff
@@ -20,5 +20,6 @@ class DefaultConfig(object):
 
     # Config variables for sending FIND-emails
     NOTIFY_FIND_API_KEY = os.getenv("NOTIFY_FIND_API_KEY")
+    NOTIFY_TEMPLATE_ID = os.getenv("NOTIFY_TEMPLATE_ID")
```

Comment:
Sorry, can we be more specific here? `NOTIFY_FIND_REPORT_DOWNLOAD_TEMPLATE_ID`? Just so you can see at a glance what it's used for. We may end up with multiple Notify template references, so just NOTIFY_TEMPLATE_ID is quite vague.